encoder/tests/defaultsInterface

```diff
@@ -35,4 +43,17 @@ export const fooDefault: Foo = {
   C: aTypeDefault,
   I1_TypedList: [1, 2],
   I2_TypedList: aTypedListDefault,
-}
\ No newline at end of file
+}
+export interface NestedFoo {
+  Bar: string;
+  Foo: {
+    Bar: string;
+    Baz: aTypeDefault;
+  };
+}
+export const nestedFooDefault: NestedFoo = {
+  Bar: 'ohai',
+}
+/**
+* The default generation for nested struct is not fully working yet, under developement...
+*/
```

Comment:
Can we make it appear before a nested default object? As though it were a doc comment on the default. And _only_ if that default object is known to have nested struct defaults that haven't been generated

Again, this is just a nice-to-have, so if it adds much effort it's better to just drop it

Reply:
you mean if there is a explicite default generated for the struct, we put the comments otherwise not? the problem for struct is today there probably has no struct has default set.

Reply:
Yeah, that's yet another reason to just ignore it for now